src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.CodeGeneration.targets

```diff
@@ -120,6 +120,18 @@
   <Target Name="_ResolveGeneratedRazorCompileInputs" BeforeTargets="ResolveRazorCompileInputs">
     <ItemGroup>
       <RazorCompile Include="@(_RazorGenerateOutput)" />
+      <RazorEmbeddedResource Include="@(RazorGenerate)" Condition="$(EmbedRazorGenerateSources)">
+        <LogicalName>/$([System.String]::Copy('%(Identity)').Replace('\','/'))</LogicalName>
+        <Type>Non-Resx</Type>
+        <WithCulture>false</WithCulture>
+      </RazorEmbeddedResource>
+    </ItemGroup>
+
+    <!-- Similar to _GenerateCompileInputs -->
+    <ItemGroup>
+      <_RazorCoreCompileResourceInputs 
+        Include="@(RazorEmbeddedResource)" 
+        Condition="'%(RazorEmbeddedResource.WithCulture)'=='false' and '%(RazorEmbeddedResource.Type)'=='Non-Resx' " />
```

Comment:
I cribbed most of this from the common.targets. Resources are complicated 😑 

Fortunately the we we used to do this is pretty simple. I'm not trying to handle any of the complicated cases for resx files and localization. This just supports using files and doing basic stuff.